pkg/apis/serving/v1alpha3/framework_onnxruntime.go

```diff
@@ -0,0 +1,26 @@
+package v1alpha3
+
+import v1 "k8s.io/api/core/v1"
+
+// ONNXRuntimeSpec defines arguments for configuring ONNX model serving.
+type ONNXRuntimeSpec struct {
+	// The location of the trained model
+	StorageURI string `json:"storageUri"`
+	// Allowed runtime versions are specified in the service config map
+	RuntimeVersion string `json:"runtimeVersion,omitempty"`
+	// Defaults to requests and limits of 1CPU, 2Gb MEM.
+	Resources v1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
Can the repeated Spec fields like `StorageURI`, `RuntimeVersion` and `Resources` be extracted out into 1 place?

Reply:
Yeah I was thinking about this myself. Resources is a bit awkward, as it's included inside "container". I'll make a FrameworkExtensions struct that we inline to the Framework specs. Similar to ComponentExtensions